corehq/ex-submodules/casexml/apps/phone/tasks.py

```diff
@@ -92,18 +94,24 @@ def update_celery_state(sender=None, headers=None, **kwargs):
 )
 def prune_synclogs():
     """
-    Drops all partition tables containing data that's older than 63 days (7 weeks)
+    Drops all partition tables containing data that's older than 63 days (9 weeks)
```

Comment:
ha! Thank you. I've seen that before and always felt weird about opening a 1 character PR.